Cargo.toml

```diff
@@ -12,16 +12,13 @@ keywords = ["proc-macro", "error", "errors"]
 categories = ["development-tools::procedural-macro-helpers"]
 license = "MIT OR Apache-2.0"
 edition = "2021"
+build = "build.rs"
 
 [dependencies]
 quote = "1"
 proc-macro2 = "1"
 proc-macro-error-attr2 = { path = "./proc-macro-error-attr", version = "=2.0.0" }
-
-[dependencies.syn]
-version = "2"
-optional = true
-default-features = false
+syn = { version = "2", optional = true, default-features = false }
```

Comment:
Nitpick: Why the diff?

Reply:
No reason - it's just clearer in my eyes. Happy to remove the change if you'd like.

Reply:
Please do.